### PR TITLE
Add Samples and api/yahoo index.ts

### PR DESCRIPTION
**Pull request by colefriedlaender**
Add Samples and api/yahoo index.ts

### CODE REVIEW

**Review by RonjaMeinecke — APPROVED** [review]
🦾GJ

**Review by cboenke — APPROVED** [review]
LGTM 🚀

File: pages/settings.tsx [hunk 1]
```diff
@@ -16,11 +15,12 @@ export default function Settings() {
       preserveAspectRatio: "xMidYMid slice",
     },
   };
+
   return (
     <div className={styles.container}>
       <Head>
         <title>MOC</title>
-        <link rel="icon" href="/Icons/favicon.png" />
+        <link rel="icon" href="/Icons/favicon.toilett.png" />
```
Comment (RonjaMeinecke) [inline pages/settings.tsx:23]: 🙌

File: pages/stocks/[id].tsx [hunk 2]
```diff
@@ -64,6 +95,18 @@ export default function Overview() {
             "Compared to the MSCI World Equity the Stock performes"
```
Comment (RonjaMeinecke) [inline pages/stocks/[id].tsx]:
```suggestion
            "Compared to the MSCI World Equity the Stock performs"
```

**Review by RonjaMeinecke — APPROVED** [review]
💥great progress

File: components/Articals/Articals.tsx [hunk 3]
```diff
@@ -13,15 +18,30 @@ function Articals({
...
   linkNameFive,
+  titleNameOne,
+  titleNameTwo,
+  titleNameThree,
+  titleNameFour,
+  titleNameFive,
 }: ArticalsProps) {
   return (
     <div className={styles.container}>
       <div className={styles.head}>Articals</div>
```
Comment (DNaunin) [inline components/Articals/Articals.tsx]: `<div className={styles.head}>Articles</div>`

**Review by DNaunin — APPROVED** [review]
LGTM! 🧻

**Pull request merged**